Feature/read timeout (#4906)
webdriverio/webdriverio: docs/Options.md

Type: `String`<br>
Default: `null`

### `connectionRetryTimeout`

> Timeout for any request to the Selenium server.

Reviewer: ```suggestion
Timeout for any WebDriver request to a driver or grid.
```
